src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -63,6 +64,11 @@ class ErgoWalletActor(settings: ErgoSettings, boxSelector: BoxSelector)
 
   private val parameters: Parameters = LaunchParameters
 
+  //todo: temporary 3.2.x collection and readers
+  private var stateReader: Option[ErgoStateReader] = None
+  private var mempoolReader: Option[ErgoMemPoolReader] = None
+  private var unspentBoxes = Seq[TrackedBox]()
```

Comment:
Please provide ScalaDoc for ErgoWalletActor with description of the general logic of how it manages the wallet state. It is hard to estimate the correctness of the changes when the general logic is not described.

Reply:
Please check PR description. Wallet logic is redone in 3.3.0, not much value in describing current one I guess.